netplugin/cluster/cluster.go

```diff
@@ -126,6 +125,34 @@ func HTTPPost(url string, req interface{}, resp interface{}) error {
 	return nil
 }
 
+// HTTPDel performs http DELETE operation
+func HTTPDel(url string) error {
+	req, err := http.NewRequest("DELETE", url, nil)
+
+	res, err := http.DefaultClient.Do(req)
+	if err != nil {
+		panic(err)
+	}
+	defer res.Body.Close()
+
+	// Check the response code
+	if res.StatusCode == http.StatusInternalServerError {
+		eBody, err := ioutil.ReadAll(res.Body)
+		if err != nil {
+			return errors.New("HTTP StatusInternalServerError" + err.Error())
+		}
+		return errors.New(string(eBody))
+	}
+
+	if res.StatusCode != http.StatusOK {
+		log.Errorf("HTTP error response. Status: %s, StatusCode: %d", res.Status, res.StatusCode)
+		return errors.New("HTTP Error response")
```

Comment:
Can this error use the same string as `log.Errorf` above it?  Would make debugging easier if anyone hits this